### DISCUSSION TITLE
2.8.2 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#4231] [讨论]
· 删除文件失败时会延时重试一次（#4203，@youzi-2333）
· 更新帮助库
　　添加 NeoForge 安装教程（LTCatt/PCL2Help#380，@WForst-Breeze）
　　删除 PCL 帮助站链接（LTCatt/PCL2Help#367，@lihl2011，PR @allMagicNB）
　　编修（LTCatt/PCL2Help#365、LTCatt/PCL2Help#376、LTCatt/PCL2Help#377，PR @wuyuncheng-26、@Fish-LP、@youzi-2333）
· 修复了以下 11 个 Bug
　　[严重] 无法安装 Beta 版 NeoForge 的整合包（#4179，PR @lactobionicAcid）
　　[严重] 自动安装无法选择部分 OptiFine（#4183，@CHlaowei，PR @lactobionicAcid）
　　获取的系统盘符可能有误（#3799，@wuliaodexiaoluo）
　　释放 JLW 失败时错误地继续了操作（#3801，@wuliaodexiaoluo）
　　无法安装 Forge 1.7.10-pre4（#4057，@3gf8jv4dv）
　　未对 { 开头的字符串正确转义（#4187，@Language-Changer）
　　快照版未显示 2.8.1 更新提示（#4189，@kong-que-dnf123）
　　错误地将 MMC 整合包识别为 CurseForge 整合包（#4194，@youzi-2333）
　　主页教学中打开画图按钮不完全泛用（#4201，@WatchfulBand）
　　开源版中点击极客蓝主题导致崩溃（#4215，@idhaname）
　　开源版编译路径是绝对路径，可能导致无法编译（#4219，PR @killcerr）

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.8.2'>2.8.2</a>.</em>

**Comment by wuliaodexiaoluo** [comment]
![QQ_1720742277827](https://github.com/user-attachments/assets/7b609af8-cdc5-43e3-9622-f5f6c0ad3b41)
![QQ_1720742293981](https://github.com/user-attachments/assets/2685baf6-0671-4cca-9105-ed14d6878308)
![QQ_1720742373713](https://github.com/user-attachments/assets/8eff5536-d512-4339-b214-39e06624238b)

~我 #3799 呢。。。。。。~

**Reply by LTCatt** [reply]
草，写漏了
